web/themes/custom/digital_gov/templates/node/node--landing-page--homepage.html.twig

```diff
@@ -0,0 +1,142 @@
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - author_picture: The node author user entity, rendered using the "compact"
+ *   view mode.
+ * - metadata: Metadata for this node.
+ * - date: (optional) Themed creation date field.
+ * - author_name: (optional) Themed author name field.
+ * - url: Direct URL of the current node.
+ * - display_submitted: Whether submission information should be displayed.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - node: The current template type (also known as a "theming hook").
+ *   - node--type-[type]: The current node type. For example, if the node is an
+ *     "Article" it would result in "node--type-article". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ *
+ * @see template_preprocess_node()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<main
+  role="main" id="main-content">
+  {# <div class="grid-container-desktop margin-bottom-5"> #}
+  <section class="usa-section" id="resources_featured">
+    <div class="tablet-lg:grid-container tablet-lg:grid-container-desktop">
+      <div class="grid-row tablet-lg:grid-gap-4">
+        <div class="grid-col-12 tablet-lg:grid-col-4">
+          <div class="box">
+            <h1 class="resources-featured__title">
+              {{ node.title.value }}
+            </h1>
+            <a class="btn btn-home" href="{{ events }}" title="View events page">
```

Comment:
These fields should pull from the link field of the landing page, `{{  content.field_buttons  }}`. Please set those fields on the homepage node and export them too. I think you should be able to do something similar to https://github.com/GSA/digital-gov-drupal/blob/9dda276b211583948ac9b0d4184c74a52456c50b/web/themes/custom/digital_gov/templates/paragraph/paragraph--external-content--featured-links.html.twig#L42, except do a for loop over the link variable.